src/commands/sessions.ts

```
@@ -0,0 +1,226 @@
+import pc from 'picocolors';
+import { loadSessionWindowData } from '../data-loader.ts';
+import { log, logger } from '../logger.ts';
+import { sharedCommandConfig } from '../shared-args.internal.ts';
+import { formatCurrency, formatNumber } from '../utils.internal.ts';
+
+export const sessionsCommand = define({
+	name: 'sessions',
+	description: 'Show usage report grouped by 5-hour session windows (Claude Max plan)',
+	...sharedCommandConfig,
+	async run(ctx) {
+		if (ctx.values.json) {
```

ESLint

[error] Unexpected any value in conditional. An explicit comparison or type conversion is required.

(ts/strict-boolean-expressions)